library/core/src/cell.rs

```diff
@@ -2159,6 +2159,7 @@ impl<T: ?Sized> UnsafeCell<T> {
     #[inline(always)]
     #[stable(feature = "rust1", since = "1.0.0")]
     #[rustc_const_stable(feature = "const_unsafecell_get", since = "1.32.0")]
+    #[cfg_attr(not(bootstrap), rustc_as_ptr)]
```

Comment:
There's also `SyncUnsafeCell::get` below, which should also be marked I think.

Reply:
Thanks. Went ahead and tagged that one as well as its raw pointer counterpart